packages/core/src/login/webview/commonAuthViewProvider.ts

```diff
@@ -158,7 +162,7 @@ export class CommonAuthViewProvider implements WebviewViewProvider {
                     <title>Base View Extension</title>
                 </head>
                 <body>
-                    <script src="https://cdnjs.cloudflare.com/ajax/libs/vue/3.4.4/vue.global.prod.min.js"></script>  
+                    <script src="${vueScript.toString()}"></script>  
```

Comment:
does the version matter? If the version changes, would it break?

Reply:
I don't think so, did local tested with the generated vsix for the amazon Q sign in and sign out, didn't see any issue for the login ux with vue.js file in `[3.3.4` which is used in the toolkit package](https://github.com/aws/aws-toolkit-vscode/blob/bfdb0ebe96213710e88461462c101808470436bf/packages/core/package.json#L576).